NEWS.md

```diff
@@ -1,5 +1,13 @@
 ## Matrix Depot Release Notes
 
+v0.7.0
+
+* Improved API (see: README.md and `?MatrixDepot`)
+* Keywrod search
```

Comment:
```suggestion
* Keyword search
```

Reply:
I changed `README.md` and some `doc/*.rst` documentation files.
In `NEWS.md` I am confused by the release numbers.
My Pkg tells me
```
(v1.0) pkg> status
    Status `/home/julia/.julia/environments/v1.0/Project.toml`
  [b51810bb] MatrixDepot v0.7.0+ [`~/.julia/dev/MatrixDepot`]
```
so I am led to think I am changing v"0.7.0" to v"0.7.1". I think it is a patch, because the
documentation in v"0.7.0" was not ok and was fixed now.

Reply:
I'm not sure what this comment refers to.

Reply:
You suggested to add "v0.8.0" to NEWS.md in a recent comment. My idea was, that after merging PR31 it was v0.7.0 and after PR32 it should be v0.7.1. If you think it should be different, please tell me to change it or just do it - and the corresponding git tags as well.

Reply:
That was in a different comment which is why I was confused. This comment was only related to a typo. I'll continue this discussion in the right thread.